sale_manual_delivery/tests/test_manual_delivery.py

```diff
@@ -412,7 +412,7 @@ def test_03_sale_multi_delivery(self):
             ' after "manual delivery" wizard call with same date',
         )
         self.assertEqual(
-            sum(first_picking.mapped("move_lines.product_uom_qty")),
+            sum(first_picking.mapped("move_line_ids.reserved_uom_qty")),
```

Comment:
Same as above, AFAIU we want to test the quantity of each move that is created, not how much was reserved.

Reply:
Hello @grindtildeath , Thank for your review. The PR has been updated.